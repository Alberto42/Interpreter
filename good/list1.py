l = [1,2,3]
l2 = [1, True, "str"]
l.append(4)
l2[1] = False
v = l2[1]
print(len(l2))
print(len(l))
print(l2[2])
print(l)
