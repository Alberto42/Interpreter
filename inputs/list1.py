l = [1,2,3]
l2 = [1, True, "str"]
l.append(4)
l2[1] = False
v = l2[1]
