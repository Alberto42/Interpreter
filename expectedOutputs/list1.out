Ident "v" = VBoolean False
Ident "l2" = VList (fromList [VInt 1,VBoolean False,VString "str"])
Ident "l" = VList (fromList [VInt 1,VInt 2,VInt 3,VInt 4])

3
4
str
[VInt 1,VInt 2,VInt 3,VInt 4]

